content/ar/docs/setup/_index.md

```diff
@@ -47,6 +47,6 @@ card:
 - اختيار [بيئة تشغيل الحاويات](/docs/setup/production-environment/container-runtimes/) لعنقودك الجديد.
 - التعرف على [أفضل الممارسات](/docs/concepts/overview/what-is-kubernetes/) لكوبرناتيز.
 
-تم تصميم كوبرناتيز {{< glossary_tooltip term_id="مستوى-التحكم" text="control plane" >}} ليعمل علي أنظمه لينكس. بإمكانك تشغيل أنظمة أخرى (مثل ويندوز) داخل العنقود بعد إعداده.
+تم تصميم كوبرناتيز {{< glossary_tooltip term_id="control-plane" text="صطح التحكم" >}} ليعمل علي أنظمه لينكس. بإمكانك تشغيل أنظمة أخرى (مثل ويندوز) داخل العنقود بعد إعداده.
```

Comment:
I still prefer the original translation مستوى-التحكم why would you prefer to change it I think you mean  سطح التحكم

Reply:
I see now that the term is entered on Wikipedia as "مستوى التحكم", so that looks good to me. All the while, "سطح التحكم" is more distinct, so I believe it has merit as well.

Reply:
Fixed-up here: [3c94187](https://github.com/kubernetes/website/pull/47820/commits/3c94187165bd9b490716b505985d42e946bf1289)